core/endpoint.go

```diff
@@ -189,6 +195,21 @@ func (endpoint *Endpoint) ValidateAndSetDefaults() error {
 	if _, contentTypeHeaderExists := endpoint.Headers[ContentTypeHeader]; !contentTypeHeaderExists && endpoint.GraphQL {
 		endpoint.Headers[ContentTypeHeader] = "application/json"
 	}
+	// Wraps the body as JSON RPC 2.0 method
+	if endpoint.JsonRPC {
+		method_name := ""
+		method_parameters := ""
+
+		if len(endpoint.Body) > 0 {
+			slices := strings.Split(endpoint.Body, " ")
+			method_name = slices[0]
+
+			if len(slices) > 1 {
+				method_parameters = strings.Join(slices[1:], ",")
+			}
+		}
+		endpoint.Body = "{\"jsonrpc\": \"2.0\", \"id\": 1, \"method\": \"" + method_name + "\", \"parameters\": [" + method_parameters + "]}\n"
```

Comment:
If this condition is not met
`if len(endpoint.Body) > 0 {`
if someone does not specify the body correctly we can have an incorrect endpoint. body. Maybe we can handle this case as error

Reply:
I thought about that. But we can send empty messages via WebSockets:

```bash
$ echo "" | websocat --jsonrpc wss://arbitrum-caddi.lavapro.xyz/one/ws
{"jsonrpc":"2.0","id":1,"error":{"code":-32600,"message":"invalid request"}}
```

What kind of error would you suggest here?